server_addon/maya/server/settings/publishers.py

```diff
@@ -299,6 +349,129 @@ class ExtractAlembicModel(BaseSettingsModel):
     families: list[str] = SettingsField(
         default_factory=list,
         title="Families")
+    autoSubd: bool = SettingsField(
+        title="Auto Subd",
+        description=(
+            "If this flag is present and the mesh has crease edges, crease "
+            "vertices or holes, the mesh (OPolyMesh) would now be written out "
+            "as an OSubD and crease info will be stored in the Alembic  file. "
+            "Otherwise, creases info won't be preserved in Alembic file unless"
+            " a custom Boolean attribute SubDivisionMesh has been added to "
+            "mesh node and its value is true."
+        )
+    )
+    dontSkipUnwrittenFrames: bool = SettingsField(
+        title="Dont Skip Unwritten Frames",
+        description=(
+            "When evaluating multiple translate jobs, this decides whether to "
+            "evaluate frames between jobs when there is a gap in their frame "
+            "ranges."
+        )
+    )
+    eulerFilter: bool = SettingsField(
+        title="Euler Filter",
+        description="Apply Euler filter while sampling rotations."
+    )
+    noNormals: bool = SettingsField(
+        title="No Normals",
+        description=(
+            "Present normal data for Alembic poly meshes will not be written."
+        )
+    )
```

Comment:
Out of curiosity - what's the difference between **No Normals** and **Write Normals**? I feel like they are both the same toggle, but the inverse?

A quick investigation shows that the argument is unused [in this function](https://github.com/ynput/ayon-core/blob/8dcda0a6b9e4f179e7a61d8f9075a6e2bb3035c4/client/ayon_core/hosts/maya/api/alembic.py#L73)

There also is no `writeNormals` in [`alembic/maya/AbcExport.cpp`](https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp).

I have a feeling that originally we had `writeNormals` inside OpenPype as the equivalent (but inverse) of `noNormals`?

As such, one of these two should be removed? (Likely the one that wasn't used before for backwards compatibility?)

Reply:
> As such, one of these two should be removed? (Likely the one that wasn't used before for backwards compatibility?)

So remove `NoNormals`?

Reply:
done